src/panels/lovelace/special-rows/hui-attribute-row.ts

```diff
@@ -57,7 +58,15 @@ class HuiAttributeRow extends LitElement implements LovelaceRow {
     return html`
       <hui-generic-entity-row .hass=${this.hass} .config=${this._config}>
         <div>
-          ${this._config.prefix} ${attribute ?? "-"} ${this._config.suffix}
+          ${this._config.prefix}
+          ${this._config.format
+            ? html` <hui-timestamp-display
+                .hass=${this.hass}
+                .ts=${new Date(attribute)}
```

Comment:
```suggestion
                .ts=${new Date(attribute.replace(" ", "T")}
```
To help iOS and macOS in case date and time separator is missing?